masterlist.yaml

```diff
@@ -2116,25 +2205,19 @@ plugins:
       - 'singleshot_retex.esp'
   - name: 'Power Armor Distribution Overhaul.esp'
     url: [ 'http://www.nexusmods.com/newvegas/mods/59677' ]
-    inc:
-      - 'boa ncrpahelmet.esp'
-      - 'Boacombat2glove.esp'
     tag:
-      - Actors.ACBS
-      - Delev
-      - Graphics
       - Invent
       - Names
-      - Relev
-      - Stats
-  - name: 'Power Armor Distribution Overhaul - Book of Steel.esp'
```

Comment:
Did you mean to get rid of the entry for `Power Armor Distribution Overhaul - Book of Steel.esp`?